crates/autopilot/src/solvable_orders.rs

```diff
@@ -283,6 +291,27 @@ impl SolvableOrdersCache {
     }
 }
 
+/// Applies fee objective scaling and discard all orders where an overflow
+/// occurred.
+fn apply_fee_objective_scaling_factor(
+    mut orders: Vec<Order>,
+    scaling_factor: &BigRational,
+) -> Vec<Order> {
+    orders.retain_mut(|order| {
+        let fee = u256_to_big_rational(&order.metadata.solver_fee);
+        let scaled_fee = fee * scaling_factor;
```

Comment:
I don't think we should scale liquidity order fees. Scaling is for artificially inflating the economic viability of a specific order, which we don't want to do for liquidity orders as it could make them be preferred over other "better" solutions over an AMM for example (by increasing the objective value more than the actual benefit they provide).

Reply:
[7b5b8ef](https://github.com/cowprotocol/services/pull/1199/commits/7b5b8ef19ef99244054b02b3c0104834e9b3ce5f)